### DISCUSSION TITLE
Game Security Violation Detected: FanControl.exe (Easy Anti-Cheat)

### DISCUSSION THREAD

**Discussion by Micke3rd** [Rem0o/FanControl.Releases#1709] [General]
wanted to share that Easy Anti-Cheat doesn't like the FanControl.exe 🙂
the following window popped up and the game was closed:

![Fancontrol caught by easy anti cheat](https://user-images.githubusercontent.com/16020794/230793850-02ea0e74-c384-4dc1-af90-941bba1913a2.png)

I assume fancontrol is using drivers with dev mode flag, doesnt it ?

**Comment by Rem0o** [comment]
This one is probably InpOut.  You can disable it in sensor settings.

**Comment by hvtim** [comment]
Maybe relevant, the same InpOut driver is also blocked by Riot Vanguard.

<img width="310" alt="image" src="https://github.com/Rem0o/FanControl.Releases/assets/16124372/506ae465-0905-4e64-bcbb-c5b5ec7a4af8">
